testing/api/test_math.py

```diff
@@ -7,7 +7,7 @@
 
 import numpy as np
 
-import spinalcordtoolbox as sct
+import spinalcordtoolbox.math as sct_math
```

Comment:
Thanks for fixing this other bug!

It must have been silently there this whole time, but the tests never failed because `math` must be imported somewhere earlier in the test suite.